pymc/tests/distributions/test_continuous.py

```diff
@@ -1802,6 +1802,46 @@ class TestTruncatedNormalUpperArray(BaseTestDistributionRandom):
     ]
 
 
+class TestZeroSumNormal(BaseTestDistributionRandom):
+    COORDS = {
+        "regions": ["a", "b", "c"],
+        "answers": ["yes", "no", "whatever", "don't understand question"],
+    }
+    with pm.Model(coords=COORDS) as m:
+        v = pm.ZeroSumNormal("v", dims=("regions", "answers"), zerosum_axes="answers")
+        s = pm.sample(10)
```

Comment:
Make this go a bit faster

```suggestion
        s = pm.sample(10, chains=1, tune=100)
```